packages/workbox-background-sync/Queue.mjs

```diff
@@ -94,6 +94,8 @@ class Queue {
     await this._runCallback('requestWillEnqueue', storableRequest);
     await this._queueStore.addEntry(storableRequest);
     await this._registerSync();
+    logger.log(`Request for '${request.url}' has been added to
```

Comment:
please wrap this in an if statement for `if (process.env.NODE_ENV !== 'production'){ ...log.... }` This allows rollup to remove the code in prod builds.

Reply:
Done